src/local/butler/reproduce.py

```diff
@@ -16,41 +16,114 @@
 from __future__ import print_function
 from builtins import object
 
+import httplib2
+import json
 import os
 import shutil
 import tempfile
+import urllib
+import webbrowser
 
+from src.python.base import utils
 from src.python.bot.tasks import commands
 from src.python.fuzzing import testcase_manager
 from src.python.system import environment
 from src.python.system import shell
 
+# TODO(mbarbella): This should work cross platform.
+AUTHORIZATION_CACHE_FILE = os.path.join(
+    os.path.expanduser('~'), '.config', 'clusterfuzz', 'authorization-cache')
 
-class _SimplifiedTestcase(object):
+# TODO(mbarbella): Don't use the old clusterfuzz-tools client id.
+OAUTH_URL = 'https://accounts.google.com/o/oauth2/v2/auth?%s' % (
+    urllib.urlencode({
+        'scope':
+            'email profile',
+        'client_id': ('981641712411-sj50drhontt4m3gjc3hordjmp'
```

Comment:
Yes, we really don't want this. As you said above, this should work without it, please find a way to get current user auth email.

Reply:
Done